platform-dom/src/main/java/com/softicar/platform/dom/elements/input/auto/DomAutoCompleteStatefulValue.java

```diff
@@ -1,11 +1,11 @@
 package com.softicar.platform.dom.elements.input.auto;
 
-class DomAutoCompleteValueAndState<T> {
+class DomAutoCompleteStatefulValue<T> {
```

Comment:
that name is confusing, I think you should rename it back

Reply:
will be re-renamed in a future PR